docs/eta-pi-p/manual.ipynb

```diff
@@ -255,24 +276,235 @@
+    "    phi2: phi23_helicity,\n",
+    "    s12: m12_square,\n",
+    "    s23: m23_square,\n",
+    "    s31: m31_square,\n",
+    "}\n",
+    "\n",
+    "Latex(aslatex(kinematic_variables))"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
+   "source": [
+    "helicity_transformer = SympyDataTransformer.from_sympy(\n",
+    "    kinematic_variables, backend=\"jax\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Parameters"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "jupyter": {
+     "source_hidden": true
+    },
+    "tags": [
+     "hide-input"
+    ]
+   },
+   "outputs": [],
    "source": [
     "a_vals = [0, 0.5, 3.5, 4, 2.5]\n",
     "b_vals = [-1.5, 4, 0.5]\n",
     "c0_val = 2.5\n",
     "m_a2_val = 1.32\n",
     "m_delta_val = 1.54\n",
     "m_nstar_val = 1.87\n",
-    "Gamma_a2_val = 0.1\n",
+    "gamma_a2_val = 0.1\n",
     "gamma_delta_val = 0.1\n",
-    "gamma_nstar_val = 0.1"
+    "gamma_nstar_val = 0.1\n",
+    "\n",
+    "parameters_default = {\n",
+    "    a[-2]: a_vals[0],\n",
+    "    a[-1]: a_vals[1],\n",
+    "    a[0]: a_vals[2],\n",
+    "    a[1]: a_vals[3],\n",
+    "    a[2]: a_vals[4],\n",
+    "    b[-1]: b_vals[0],\n",
+    "    b[0]: b_vals[1],\n",
+    "    b[1]: b_vals[2],\n",
+    "    c0: c0_val,\n",
+    "    m_a2: m_a2_val,\n",
+    "    m_delta: m_delta_val,\n",
+    "    m_nstar: m_nstar_val,\n",
+    "    gamma_a2: gamma_a2_val,\n",
+    "    gamma_delta: gamma_delta_val,\n",
+    "    gamma_nstar: gamma_nstar_val,\n",
+    "}\n",
+    "\n",
+    "Latex(aslatex(parameters_default))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Phase Space Generation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Mass for $p\\gamma$ system"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "E_lab_gamma = 8.5\n",
+    "m_proton = 0.938\n",
+    "m_0 = np.sqrt(2 * E_lab_gamma * m_proton + m_proton**2)\n",
+    "m_eta = 0.548\n",
+    "m_pi = 0.135\n",
+    "m_0"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rng = TFUniformRealNumberGenerator(seed=0)\n",
+    "phsp_generator = TFPhaseSpaceGenerator(\n",
+    "    initial_state_mass=m_0,\n",
+    "    final_state_masses={1: m_eta, 2: m_pi, 3: m_proton},\n",
+    ")\n",
+    "phsp_momenta = phsp_generator.generate(500_000, rng)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pd.DataFrame({\n",
+    "    (k, label): np.transpose(v)[i]\n",
+    "    for k, v in phsp_momenta.items()\n",
+    "    for i, label in enumerate([\"E\", \"px\", \"py\", \"pz\"])\n",
+    "})"
```

Comment:
I would remove `pandas` and this cell. Looks fancy, but the values don't tell the reader much.